modules/account-lib/test/unit/coin/dot/transactionBuilder/base.ts

```diff
@@ -0,0 +1,185 @@
+import { BaseCoin as CoinConfig, coins } from '@bitgo/statics';
+import { UnsignedTransaction } from '@substrate/txwrapper-core';
+import should from 'should';
+import sinon, { assert } from 'sinon';
+import { TransactionType } from '../../../../../src/coin/baseCoin';
+import { BaseKey } from '../../../../../src/coin/baseCoin/iface';
+import { specNameType } from '../../../../../src/coin/dot/iface';
+import { Transaction } from '../../../../../src/coin/dot/transaction';
+import { TransactionBuilder } from '../../../../../src/coin/dot/transactionBuilder';
+import * as DotResources from '../../../../resources/dot';
+
+class StubTransactionBuilder extends TransactionBuilder {
```

Comment:
Instead of implement a Stub, Could we create a theory test for each builder that extended it?

Reply:
I dont understand you here